src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -149,6 +150,15 @@ internal static partial class Ssl
         [GeneratedDllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_SslSetData")]
         internal static partial int SslSetData(IntPtr ssl, IntPtr data);
 
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_SslUseCertificate")]
+        internal static extern int SslUseCertificate(SafeSslHandle ssl, SafeX509Handle certPtr);
+
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_SslUsePrivateKey")]
+        internal static extern int SslUsePrivateKey(SafeSslHandle ssl, SafeEvpPKeyHandle keyPtr);
+
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_SslSetClientCertCallback")]
+        internal static extern unsafe void SslSetClientCertCallback(SafeSslHandle ssl, int set);
```

Comment:
OpenSSL exposes two very similar functions: `SSL_CTX_set_client_cert_cb` and `SSL_set_cert_cb`, as far as I can tell, the code uses the latter one, but is named like the former one, which may be potentially confusing.

Reply:
I mentioned the slow re-factor as I'm trying to make more clear use of SSL_CTX vs SSL. 
For all the new functions I'm following naming of SslCtxSetXXX for what operates on CTX and SslSetXXX for the one on SSL. There may be still some code that does not work that way but that is my general intention.

https://github.com/dotnet/runtime/blob/2a86305d8503d17284e0f32a18d5e2d06d9275ee/src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.SslCtx.cs#L22-L26

Reply:
Specifically, I meant the "Client" part of the name, but since this function does not actually accept a callback from managed code, I no longer think that is an issue.